### DISCUSSION TITLE
WiFi does not work anymore after macOS 13.2.1 update with MacBook Pro 10.1

### DISCUSSION THREAD

**Discussion by mitchtokita** [community/community#47953] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

After updating my Macbook pro 10.1 from 13.2 to 13.2.1 the WiFi stopped working. Everything looks regular but the internet doesn't work at all. I have a bootcamp installation of windows where WiFi works normal.

**Comment by jge162** [comment]
make sure you are using the correct setting for the wife and router to mate. 802.11a, 802.11b/g/n, and/or 802.11ac wireless standards . maybe by upgrading you changed the firmware and your router is broadcasting an option you cannot connect to? or who really knows trying going somewhere else and try to connect or try a personal hotspot. bests

**Comment by queenofcorgis** [comment]
Hi @mitchtokita,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as your WiFi provider. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
